### PR TITLE
Improve DKG & QC contract logging

### PR DESCRIPTION
**Pull request by kc1116**
This PR updates the DKG and QC contract client logger initialization to include the node ID of the AN that the flow client being used is connecting to.

### CODE REVIEW

File: module/dkg/client.go [hunk 1]
```diff
@@ -33,13 +33,17 @@ type Client struct {
...
 	signer sdkcrypto.Signer,
 	dkgContractAddress,
 	accountAddress string,
 	accountKeyIndex uint,
 ) *Client {
 
-	log = log.With().Str("component", "dkg_contract_client").Logger()
+	log = log.With().
+		Str("component", "dkg_contract_client").
+		Str("flow-client-an-id", flowClientANID.String()).
```
Comment (jordanschalm) [inline module/dkg/client.go:45]:
```suggestion
		Str("flow_client_an_id", flowClientANID.String()).
```
Nitpick: use same casing as above

File: module/epochs/qc_client.go [hunk 2]
```diff
@@ -42,14 +42,18 @@ type QCContractClient struct {
...
 	accountAddress string,
 	accountKeyIndex uint,
 	qcContractAddress string,
 	signer sdkcrypto.Signer,
 ) *QCContractClient {
 
-	log = log.With().Str("component", "qc_contract_client").Logger()
+	log = log.With().
+		Str("component", "qc_contract_client").
+		Str("flow-client-an-id", flowClientANID.String()).
```
Comment (jordanschalm) [inline module/epochs/qc_client.go:55]: same nitpick here

**Review by jordanschalm — APPROVED** [review]
Good idea, thanks for adding this

**Review by zhangchiqing — APPROVED** [review]

File: module/epochs/qc_client.go (hunk 2 above)
Comment (kc1116) [reply module/epochs/qc_client.go]: https://github.com/onflow/flow-go/pull/2263/commits/3b67687eae3a7501cf779878248295092a0dc604

File: module/dkg/client.go (hunk 1 above)
Comment (kc1116) [reply module/dkg/client.go]: https://github.com/onflow/flow-go/pull/2263/commits/3b67687eae3a7501cf779878248295092a0dc604

**Comment by kc1116** [comment]
bors merge

**Pull request merged**